solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/Modules/ActivityDetails/ActivityDetailsViewModelTests.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ActivityDetailsViewModel.swift
+//  FinanceAppTests
+//
+//  Created by Leonardo Almeida on 22/08/22.
+//
+
+import Foundation
+import XCTest
+
+@testable import FinanceApp
+
+final class ActivityDetailsViewModelTests: XCTestCase {
+    
+    let delegate = ActivityDetailsViewModelDelegateStub()
+    
+    func test_WhenFecthData_ShouldReturnCountOne() {
+        // GIVEN
+        let financeServiceStub = FinanceServiceStub()
+        
+        var activityViewModelStub = ActivityDetailsViewModel(financeService: financeServiceStub)
+        activityViewModelStub.delegate = delegate
+        
+        // WHEN
+        activityViewModelStub.fetchData()
+        
+        let expectation = expectation(description: "test_WhenFecthData_ShouldReturnCountOne")
```

Comment:
[sugestao] normalmente na expectation, escolhemos algo mais legível, como "expected to get [something]"
[pergunta] por que precisamos teste assíncrono aqui? Uma boa pratica e converter qualquer código assíncrono em síncrono.
O nosso mock/stub não precisa ir pra internet e baixar algum dado, nos imediatamente vamos dar o que e necessário; dessa maneira, acho que talvez não precisemos de testes assíncronos aqui.
Ou se assincrono e necessario (e o teste falha se tiver um teste síncrono), eu diria que talvez milésimos de segundos seja o suficiente.